### PR TITLE
Document different installation methods

### PR DESCRIPTION
**Pull request by jrj-d**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,63 @@
...
+
+The diagram below details those different installation methods.
+Read on for more details.
+
+![Terraform module usage](docs/module_usage.svg)
+
+### 1. Create an Ocean Spark cluster from scratch
+
+In this case, use the [`ocean-eks` Terraform module](https://registry.terraform.io/modules/spotinst/ocean-eks/spotinst/latest) to create an Ocean cluster from scratch.
+Then, use the the `ocean-spark` Terraform module (this module) to import the cluster into Ocean Spark.
```
Comment (jystephan) [inline README.md:23]: nit: the the

File: README.md [hunk 2]
```diff
@@ -1,28 +1,63 @@
...
+
+Folder [`examples/eks-ocean/`](examples/eks-ocean/) contains a full example.
+
+### 2. Import an existing EKS cluster
+
+In this case, use the [`ocean-controller` Terraform module](https://registry.terraform.io/modules/spotinst/ocean-controller/spotinst/latest) and the [`spotinst_ocean_aws` Terraform resource](https://registry.terraform.io/providers/spotinst/spotinst/latest/docs/resources/ocean_aws) to import the EKS cluster into Ocean.
+Then, use the the `ocean-spark` Terraform module (this module) to import the resulting Ocean cluster into Ocean Spark.
+
+Folder [`examples/foo-bar/`](examples/foo-bar/) contains a full example.
```
Comment (jystephan) [inline README.md:32]: Reminder to update url. Also I wouldn't say "full example", as our example covers the second step.

File: README.md [hunk 3]
```diff
@@ -1,28 +1,63 @@
...
+Read on for more details.
+
+![Terraform module usage](docs/module_usage.svg)
+
+### 1. Create an Ocean Spark cluster from scratch
+
+In this case, use the [`ocean-eks` Terraform module](https://registry.terraform.io/modules/spotinst/ocean-eks/spotinst/latest) to create an Ocean cluster from scratch.
+Then, use the the `ocean-spark` Terraform module (this module) to import the cluster into Ocean Spark.
+
+Folder [`examples/eks-ocean/`](examples/eks-ocean/) contains a full example.
```
Comment (jystephan) [inline README.md:25]: I wouldn't say that it contains a full example. It contains an example of the second step. The first step is on ocean-eks.

**Review by jystephan — APPROVED** [review]
Looks great, minor comments inlined!

File: README.md [hunk 4]
```diff
@@ -1,28 +1,63 @@
...
+A Terraform module to install the [Ocean for Apache Spark](https://spot.io/products/ocean-apache-spark/) data platform.
 
-### :warning: Ocean for Apache Spark is currently only available on AWS
+### :warning: Ocean for Apache Spark is currently only available on AWS.
+
+## Overview
+
+This module imports an existing Ocean cluster into Ocean Spark.
+It can be combined with other Ocean Terraform modules to support a number of installation methods for Ocean Spark:
+1. create an Ocean Spark cluster from scratch in your cloud account
```
[the remark this answers is not in the document]
Comment (jrj-d) [reply README.md:11]: Thanks!

File: README.md (hunk 1 above)
Comment (jrj-d) [reply README.md:23]: thanks!

File: README.md [hunk 5]
```diff
@@ -1,28 +1,63 @@
...
+In this case, use the [`ocean-controller` Terraform module](https://registry.terraform.io/modules/spotinst/ocean-controller/spotinst/latest) and the [`spotinst_ocean_aws` Terraform resource](https://registry.terraform.io/providers/spotinst/spotinst/latest/docs/resources/ocean_aws) to import the EKS cluster into Ocean.
+Then, use the the `ocean-spark` Terraform module (this module) to import the resulting Ocean cluster into Ocean Spark.
+
+Folder [`examples/foo-bar/`](examples/foo-bar/) contains a full example.
+
+### 3. Import an existing Ocean cluster
+
+In this case, use the the `ocean-spark` Terraform module (this module) to import the existing Ocean cluster into Ocean Spark.
+
+Folder [`examples/exisiting-ocean-cluster/`](examples/exisiting-ocean-cluster/) contains a full example.
```
[the remark this answers is not in the document]
Comment (jrj-d) [reply README.md:38]: Yes I'll fix the examples folder and change file names when @ImpSy has pushed the new example

File: README.md (hunk 3 above)
Comment (jrj-d) [reply README.md:25]: For each installation method, we provide a full example (in this case, from scratch to Ocean Spark). If we were to show only the second step (import the cluster into Ocean Spark), then we could have a single example for all three installation methods. The second step is shared.

**Pull request merged**